### ISSUE TITLE
Set default contact name to current user?

### ISSUE THREAD

**Issue by kosack** [cta-observatory/ctapipe#1761]
We could add a default contact name to DataWriter in case the user doesn't specify one in a config file that just gives the current user's full name, e.g.:
```py3
import os
import pwd
default = pwd.getpwuid(os.getuid()).pw_gecos
```
That would make it less error-prone track the provenance of files created by users.  In productions, the config file should set it properly however. This info (well only the username, not full name) is already in the provenance.log, but not in the reference metadata in the file itself. 

not sure if that works on Windows, but we don't really support windows.

**Comment by maxnoe** [comment]
> This module provides access to the Unix user account and password database. It is available on all Unix versions.

**Comment by kosack** [comment]
I suppose we can just wrap it in a `try:catch`, and if it works, great, if not on windows or similar, it's not a big issue.   It's best to set a good default if possible, otherwise fall back to None. The idea is only to make it less error-prone if the  user forgets to fill in the info.

**Comment by kosack** [comment]
this may be redundant if #1774 is merged - it forces the user to provide the contact info in the generated config files

**Comment by kosack** [comment]
implemented in #1774
